### PR TITLE
Add swagger documentation for /spec endpoint

### PR DESCRIPTION
**Pull request by guillett**
#### New features

- /spec endpoint is described in the swagger documentation

### CODE REVIEW

File: openfisca_web_api_preview/openAPI.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ tags:
       description: "Variable official documentation"
       url: "http://openfisca.org/doc/variables.html"
   - name: "Calculations"
+  - name: "Docu"
```
Comment (fpagnoux) [inline openfisca_web_api_preview/openAPI.yml]: Missing `mentation` ?

Further you seem to refer to a tag name `Documentation `

File: openfisca_web_api_preview/openAPI.yml [hunk 2]
```diff
@@ -169,6 +170,19 @@ paths:
           description: "The request is invalid. Details about the error are sent back in the response body"
           headers:
             $ref: '#/commons/Headers'
+  /spec:
+    get:
+      summary: "Provide the API documentation in a OpenAPI format"
```
Comment (fpagnoux) [inline openfisca_web_api_preview/openAPI.yml]: > an OpenApi format 

Google has more result for "an OpenApi" than "a OpenApi"

**Comment by guillett** [comment]
Ping @fpagnoux :-)

**Comment by maukoquiroga** [comment]
Rebased

**Pull request merged**